.github/workflows/codespace-test.yml

```
@@ -0,0 +1,40 @@
+    - uses: actions/checkout@v3
+    
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+    
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)